sdk-base/src/main/java/com/mapbox/maps/extension/style/StyleContract.kt

```diff
@@ -135,4 +140,16 @@ interface StyleContract {
      */
     fun bindTo(delegate: StyleInterface)
   }
+
+  /**
+   * Define the interfaces for the model extension.
+   */
+  fun interface StyleModelExtension {
```

Comment:
as this is public interface - should be marked as experimental

Reply:
on the other hand API here will not change for sure, so perhaps useless to mark it in such a manner

Reply:
good catch, I think we should mark this one as experimental, and remove it together with other model layer related APIs at the same time.